openpathsampling/tests/test_helpers.py

```diff
@@ -306,18 +304,18 @@ def _wrapper(*args, **kwargs):
     return decorator
 
 def assert_frame_equal(truth, beauty):
-    assert_equal(len(truth.index), len(beauty.index))
-    assert_equal(len(truth.columns), len(beauty.columns))
-    assert_equal(set(truth.index), set(beauty.index))
-    assert_equal(set(truth.columns), set(beauty.columns))
+    assert len(truth.index) == len(beauty.index)
+    assert len(truth.columns) == len(beauty.columns)
+    assert set(truth.index) == set(beauty.index)
+    assert set(truth.columns) == set(beauty.columns)
     for idx in truth.index:
         for col in truth.columns:
             truth_val = truth.loc[idx, col]
             beauty_val = beauty.loc[idx, col]
             if np.isnan(truth_val):
-                assert_true(np.isnan(beauty_val))
+                assert np.isnan(beauty_val)
```

Comment:
Huh, I never knew `assert_true` accepted `truthy` values, I thoughed it checked for `is True`.

But looking at the code for `assert_true`
```python
   def assertTrue(self, expr, msg=None):                                       
        """Check that the expression is true."""                                
        if not expr:                                                            
            msg = self._formatMessage(msg, "%s is not true" % safe_repr(expr))  
            raise self.failureException(msg)                                    
```

 this is indeed the correct replacement.